src/invidious/yt_backend/youtube_api.cr

```diff
@@ -500,7 +500,11 @@ module YoutubeAPI
       data["params"] = params
     end
 
-    return self._post_json("/youtubei/v1/player", data, client_config)
+    if !CONFIG.invidious_companion.empty?
+      return self._post_invidious_companion("/youtubei/v1/player", data)
```

Comment:
Just as a side note here: because we've a dedicated function to call the companion, we're not forced to use the same URL path as youtube! The path can even be hardcoded (= one less argument required), as we're sure it will never change.